### PR TITLE
[5.3] Simplify conditional

### PR DESCRIPTION
**Pull request by jerguslejko**
This little PR simplifies the conditional in `ModelMakeCommand.php`. 🌵

### CODE REVIEW

File: src/Illuminate/Foundation/Console/ModelMakeCommand.php [hunk 1]
```diff
@@ -36,24 +36,26 @@ class ModelMakeCommand extends GeneratorCommand
      */
     public function fire()
     {
-        if (parent::fire() !== false) {
-            if ($this->option('migration')) {
-                $table = Str::plural(Str::snake(class_basename($this->argument('name'))));
+        if (parent::fire() == false) {
```
Comment (fernandobandeira) [inline src/Illuminate/Foundation/Console/ModelMakeCommand.php]: It was ``` !== ``` previously so i guess this should be ``` === ```.

**Comment by jerguslejko** [comment]
@fernandobandeira ah, yes! you are right. I've change it

**Comment by browner12** [comment]
i don't want to labor on this since both before and after do the same thing. but i'm curious why this simplifies it? you just moving the code from inside to the outside of the conditional?  is it purely a readability thing?

**Comment by jerguslejko** [comment]
Yeah, it's only meant to improve readability

**Pull request merged**